aws-glue-datacatalog-client-common/src/main/java/com/amazonaws/glue/catalog/util/BatchCreatePartitionsHelper.java

```diff
@@ -27,7 +23,7 @@
 
   private static final Logger logger = Logger.getLogger(BatchCreatePartitionsHelper.class);
 
-  private final AWSGlue client;
+  private final AWSGlueMetastore glueClient;
```

Comment:
shall we rename variable name as glueMetaStore rather client, it creates confusion as we referring glue sdk client.